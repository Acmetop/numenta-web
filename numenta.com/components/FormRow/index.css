/**
 * Numenta.com company website source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.formRow {
  composes: cb from 'tachyons';
  composes: cf from 'tachyons';
  composes: flex from 'tachyons';
  composes: mb2 from 'tachyons';
  composes: mr4 from 'tachyons';
  flex-direction: row;
}
