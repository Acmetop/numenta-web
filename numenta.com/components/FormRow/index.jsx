// Numenta.com company website source code
// MIT License (see LICENSE.txt)
// Copyright © 2005—2016 Numenta <http://numenta.com>

import React from 'react'

import styles from './index.css'


const FormRow = ({children}) => (
  <div className={styles.formRow}>
    {children}
  </div>
)

FormRow.propTypes = {
  children: React.PropTypes.node.isRequired,
}

export default FormRow
