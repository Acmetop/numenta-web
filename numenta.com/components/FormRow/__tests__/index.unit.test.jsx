import React from 'react'
import renderer from 'react-test-renderer'

import FormRow from '../../FormRow'


describe('<FormRow /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <FormRow />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
