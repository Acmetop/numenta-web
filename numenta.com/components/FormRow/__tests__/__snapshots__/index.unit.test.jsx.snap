exports[`<FormRow /> React component unit test suite Renders correctly 1`] = `
<div
  className="formRow" />
`;
